plugins-compat-tester-cli/src/main/java/org/jenkins/tools/test/CliOptions.java

```diff
@@ -127,6 +132,10 @@
     @Parameter(names="-help", description = "Print this help message")
     private boolean printHelp;
 
+    @Parameter(names = "-overridePlugins", description = "List of plugins to use to test a plugin in place of the normal dependencies." +
+          "Format: 'PLUGIN_NAME=PLUGIN_VERSION", converter = PluginConverter.class, validateWith = PluginValidator.class)
+    private List<Plugin> overridePlugins;
```

Comment:
* I think you should initialize it to an empty list or something. Null list are generally an unnecessary pain to endure.
* rename this to `overridenPlugins`?